pages/common/git-tag.md

```diff
@@ -0,0 +1,19 @@
+# git tag
+
+> Create, list, delete or verify tags
```

Comment:
A 3-4 word description of what a tag is (e.g. "immutable references to specific commits") would be nice here, in brackets.